src/adapters/search_history.js

```diff
@@ -0,0 +1,40 @@
+import { get, set, del } from './store';
+
+const SEARCH_HISTORY_KEY = 'search_history';
+const HISTORY_SIZE = 100;
+
+// Add a query to the list
+export function save_query(q) {
```

Comment:
This is still snake case… and I know why I thought we had a camelCase rule for ESLint, it's in Phoenix.
I'll take a note to add the same.